modules/openideal_slideshow/js/slider-configuration.js

```diff
@@ -0,0 +1,294 @@
+(function ($, Drupal) {
+
+  /**
+   * Init the photoswipe slider.
+   *
+   * @param {string} gallerySelector
+   *   The index of the current element.
+   * @param {object} mySwiper
+   *   The slick HTML element.
+   */
+  var initPhotoSwipeFromDOM = function (gallerySelector, mySwiper) {
```

Comment:
1) Please add a link to the original example: https://photoswipe.com/documentation/getting-started.html
2) Please move custom Drupal behavior to the top of the file.